src/meta/meta_ckpt.c

```diff
@@ -859,6 +859,44 @@ __wt_ckpt_blkmod_to_meta(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_CKPT *ckpt)
     return (0);
 }
 
+/*
+ * __wt_meta_ckptlist_update_config --
+ *     Provided a metadata config, set a file's checkpoint value from the WT_CKPT list.
```

Comment:
We generally avoid referencing struct names ```WT_CKPT``` in comments, so that in future if we change the struct name, we don't forget changing in the comments. Maybe we can just say "checkpoint list". I don't feel strongly about this, so feel free to ignore this comment.

Reply:
Updated comment in 46454c5 - thanks!